.github/workflows/release.yaml

```diff
@@ -0,0 +1,84 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # fetch all history so we can get the latest tag
+          fetch-depth: 0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push images
+        run: |
+          IMAGE_PREFIX="ghcr.io/gke-labs/gemini-for-kubernetes-development"
+          TAG=${{ github.ref_name }}
+          docker build -t "${IMAGE_PREFIX}/github-mcp-server:${TAG}" -f images/github-mcp-server/Dockerfile repo-agent/
+          docker build -t "${IMAGE_PREFIX}/issue-sidecar:${TAG}" -f images/issue-sidecar/Dockerfile repo-agent/
+          docker build -t "${IMAGE_PREFIX}/configdir-controller:${TAG}" -f images/configdir-controller/Dockerfile repo-agent/
+          docker build -t "${IMAGE_PREFIX}/review-sidecar:${TAG}" -f images/review-sidecar/Dockerfile repo-agent/
+          docker build -t "${IMAGE_PREFIX}/review-ui:${TAG}" -f images/review-ui/Dockerfile repo-agent/
+          docker build -t "${IMAGE_PREFIX}/repowatch-controller:${TAG}" -f images/repowatch-controller/Dockerfile repo-agent/
+          docker build -t "${IMAGE_PREFIX}/review-api:${TAG}" -f images/review-api/Dockerfile repo-agent/
+          docker build -t "${IMAGE_PREFIX}/redis:${TAG}" -f images/redis/Dockerfile repo-agent/
+          docker build -t "${IMAGE_PREFIX}/review-sandbox:${TAG}" -f images/review-sandbox/Dockerfile repo-agent/
+          docker build -t "${IMAGE_PREFIX}/issue-sandbox:${TAG}" -f images/issue-sandbox/Dockerfile repo-agent/
+          docker build -t "${IMAGE_PREFIX}/configdir-cli:${TAG}" -f images/configdir-cli/Dockerfile repo-agent/
+          docker push "${IMAGE_PREFIX}/github-mcp-server:${TAG}"
+          docker push "${IMAGE_PREFIX}/issue-sidecar:${TAG}"
+          docker push "${IMAGE_PREFIX}/configdir-controller:${TAG}"
+          docker push "${IMAGE_PREFIX}/review-sidecar:${TAG}"
+          docker push "${IMAGE_PREFIX}/review-ui:${TAG}"
+          docker push "${IMAGE_PREFIX}/repowatch-controller:${TAG}"
+          docker push "${IMAGE_PREFIX}/review-api:${TAG}"
+          docker push "${IMAGE_PREFIX}/redis:${TAG}"
+          docker push "${IMAGE_PREFIX}/review-sandbox:${TAG}"
+          docker push "${IMAGE_PREFIX}/issue-sandbox:${TAG}"
+          docker push "${IMAGE_PREFIX}/configdir-cli:${TAG}"
+
+      - name: Run deployment script
+        run: |
+          chmod +x ../dev/tools/deploy-to-kube
```

Comment:
Does the script use ./repo-agent as the root or the root dir of this repo?  Might need to update the `../dev/tools/deploy-to-kube` path or change the workdir IIUC

Reply:
fixed